src/index.html

```diff
@@ -3,12 +3,57 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css?family=Font+Name" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+  <header class="header">
+    <a href="#" class="logo">
+      <img src="./images/logo.png" alt="Moyo logo"
+      class="logo__image">
```

Comment:
Need to fix it everywhere
```suggestion
      <img
        src="./images/logo.png"
        alt="Moyo logo"
        class="logo__image"
      >
```